test/test.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+require('bedrock-https-agent');
+const path = require('path');
+
+const {asyncHandler, middleware: {acceptableContent}} =
+  require('bedrock-express');
+
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-express.configure.static', () => {
+  bedrock.config.express.static.push({
+    route: '/static/foo',
+    path: path.join(__dirname, 'static', 'foo.html'),
+    file: true,
+    cors: true,
+  });
+  bedrock.config.express.static.push({
+    route: '/static/',
```

Comment:
So we don't usually configure routes using `/` on the end. Can this be changed to match convention?

Reply:
Changed caa4e1b